src/test/test-chase.c

```diff
@@ -202,23 +204,24 @@ TEST(chase) {
 
```

Comment:
Please change `ASSERT_OK()` -> `ASSERT_OK_ERRNO()` for mkdir(), symlink(), chmod().
Also, please use `ASSERT_ERROR(chase(...), ENOLINK);`